src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -317,6 +319,107 @@ void RenderTarget::draw(const Vertex* vertices, std::size_t vertexCount,
 
 
 ////////////////////////////////////////////////////////////
+void RenderTarget::draw(const VertexBuffer& vertexBuffer, const RenderStates& states)
+{
+    draw(vertexBuffer, 0, vertexBuffer.m_size, states);
+}
+
+
+////////////////////////////////////////////////////////////
+void RenderTarget::draw(const VertexBuffer& vertexBuffer, std::size_t firstVertex, std::size_t vertexCount, const RenderStates& states)
+{
+    // VertexBuffer not supported?
+    if (!VertexBuffer::isAvailable())
+    {
+        err() << "sf::VertexBuffer is not available, drawing skipped" << std::endl;
+        return;
+    }
+
+    // Sanity check
+    if (firstVertex > vertexBuffer.m_size)
+        return;
+
+    // Clamp vertexCount to something that makes sense
+    vertexCount = std::min(vertexCount, vertexBuffer.m_size - firstVertex);
+
+    // Nothing to draw?
+    if (!vertexCount || !vertexBuffer.m_buffer)
+        return;
+
+    // GL_QUADS is unavailable on OpenGL ES
+    #ifdef SFML_OPENGL_ES
+        if (vertexBuffer.m_primitiveType == Quads)
+        {
+            err() << "sf::Quads primitive type is not supported on OpenGL ES platforms, drawing skipped" << std::endl;
+            return;
+        }
+        #define GL_QUADS 0
```

Comment:
Won't we have a preprocessor warning about GL_QUADS being defined twice (the other one being in `RenderTarget::draw(Vertex*)), when compiling with OpenGL ES?

Reply:
CI didn't seem to complain about it... doesn't mean other compilers won't though :P.